docker-storage-setup.1

```diff
@@ -12,11 +12,17 @@ docker\-storage\-setup - Grows the root filesystem and sets up storage for docke
 
 \f[B]--reset\f[]
   Reset your docker storage to init state. Reset does not try to remove volume groups or try to remove any of the disks added previously.
-  
+
 Note: The
 \f[B]--reset\f[]
 command is not sufficient to cleanup your docker environment.  This option is used by other tools. (\f[B]atomic storage reset\f[])
 
+\f[B]--dump-config\f[]
+  Write part of the configuration to stdout in a machine readable
+  format.  Note that this does not describe the current Docker
```

Comment:
Let these just be key=value pairs and these should be readable by humans as well as other programs. No need to call it machine readable.